README.md

```diff
@@ -49,6 +49,10 @@ config:
     - content: "force-unsafe-io\n"
       path: /etc/dpkg/dpkg.cfg
       append: true
+    # wait for cloud-init to be done
+    - content: "#!/bin/sh\nexec cloud-init status --wait --long\n"
```

Comment:
Can we call it cloud-init-wait to avoid confusion with "ci" in this repo meaning "continuous integration"?

Reply:
As there's no common need to type this command I would argue clarity in the script is of more value here than shorter overloaded acronyms.